src/main/java/io/cryostat/Cryostat.java

```diff
@@ -79,8 +81,20 @@ public static void main(String[] args) throws Exception {
         client.credentialsManager().load();
         client.ruleRegistry().loadRules();
         client.ruleProcessor().enable();
-        client.httpServer().start();
-        client.webServer().start();
+        client.vertx()
+                .deployVerticle(
+                        client.httpServer(),
+                        new DeploymentOptions().setWorker(false),
+                        res -> {
+                            System.out.println("HTTP Server Verticle Started");
+                        });
+        client.vertx()
+                .deployVerticle(
+                        client.webServer(),
+                        new DeploymentOptions().setWorker(true),
```

Comment:
The `WebServer` is deployed as a `Worker Verticle` here because we don't want the HTTP request handlers to be doing their work on an event loop thread, correct?

Reply:
Yep correct, we want the request handlers on worker threads.

Reply:
The `BlockingHandlerDecorator` is a key piece of what's actually putting work onto worker threads too, btw. But from my experimentation here that only seemed to work correctly when the WebServer was deployed as a worker verticle.

Reply:
I also just realized the naming is a little unfortunate. The `BlockingHandler` I added is the `/api/beta/block` API endpoint that blocks 5 seconds and then responds "hello", named because it blocks. `BlockingHandlerDecorator` is a Vertx class that wraps around a handler definition and perform its work on a worker thread then enqueues the next handler in series, named because it "decorates" (wraps) around handlers that block and lets them behave nice and async.